mobsf/MobSF/views/saml2.py

```diff
@@ -174,6 +183,9 @@ def saml_acs(request):
                 'role attribute not found in SAML response.')
         email = attributes['email'][0]
         role = get_user_role(attributes['role'])
+        if not role:
+            raise Exception(
```

Comment:
Users/Groups that doesn't need MobSF access shouldn't be added in the MobSF SAML app.
https://mobsf.github.io/docs/#/sso